CONTRIBUTING.md

```diff
@@ -435,6 +436,17 @@ ghstack submit
 [`ghstack`](https://github.com/ezyang/ghstack). It creates a large commit that is
 of very low signal to reviewers.
 
+## Merging your Change
+If you know the right people or team that should approve your PR, add them to the Reviewers list.
```

Comment:
One needs to have special permissions to add reviews.

Reply:
good to know! updated accordingly